.prettierrc

```diff
@@ -1,8 +1,11 @@
 {
+  "singleQuote": true,
+  "trailingComma": "all",
   "overrides": [
     {
       "files": "*.sol",
       "options": {
+        "singleQuote": false,
```

Comment:
Do we need these changes to `.prettierrc` ? Can you explain what they do exactly ?

Reply:
The lint config doesn't match the prettier config for what I've noticed using the vscode prettier extension (although running the package.json lint scripts would give the same result)